### PR TITLE
feat: deliver B-allele plot

### PR DESCRIPTION
**Pull request by ivadym**
### This PR:

Made available B-allele plot (produced by ascatNGS) for Scout.

### Review and tests: 
- [ ] Tests pass
- [ ] Code review
- [ ] New code is executed and covered by tests, and test approve

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -25,6 +25,7 @@ Added:
 * Zenodo integration to create citable link #813
 * Panel BED specific QC conditions #800
 * Metric extraction to a YAML file for Vogue #802
+* ``ascatNGS`` tumor normal delivery #810
```
Comment (ashwini06) [inline CHANGELOG.rst:28]: Add this under the new release [X.X.X]

Comment (ivadym) [reply CHANGELOG.rst:28]: Is this a minor change or should be consider as a patch?

Comment (khurrammaqbool) [reply CHANGELOG.rst:28]: I suggest minor change.

**Review by khurrammaqbool — APPROVED** [review]
Please add the output/log if possible.

File: CHANGELOG.rst (hunk 1 above)
Comment (ashwini06) [reply CHANGELOG.rst:28]: A new set of deliverables are added. I think should be a minor.

Comment (ivadym) [reply CHANGELOG.rst]: 👍

**Comment by ivadym** [comment]
Tested on hasta:
Screen Shot 2021-11-23 at 17 24 56

**Review by hassanfa — COMMENTED** [review]
Similar to PR #820

Regarding the version bump:
- It should be a patch since there is no change in the workflow, output files, or structure of workflow.
- If you're making a PR to develop, please change the version to "[X.X.X]"

**Pull request merged**